lib/shoryuken/queue.rb

```diff
@@ -52,6 +49,21 @@ def fifo?
 
     private
 
+    def set_name_and_url(name_or_url)
+      if name_or_url.start_with?('https://sqs.')
```

Comment:
In amazon's docs I only see references to the ones with the http(s), but I'm using a front end for it called Convox that gives me queues URI's that start with sqs://.  I'm not sure about the implications of that.  I wonder if this is just a convox specific thing...  Interestingly enough, they can be accessed by changing the sqs:// to https:// so what you're proposing would work as long as everyone knows to do that.